config/admin.config.php

```diff
@@ -61,70 +59,70 @@
+            'ZF\Apigility\Doctrine\Admin\Model\DoctrineRpcServiceModelFactory'  => DoctrineRpcServiceModelFactory::class,
+            'ZF\Apigility\Doctrine\Admin\Model\DoctrineRpcServiceResource'      => DoctrineRpcServiceResource::class,
         ],
         'factories' => [
-            Model\DoctrineAutodiscoveryModel::class      => Model\DoctrineAutodiscoveryModelFactory::class,
-            Model\DoctrineMetadataServiceResource::class => Model\DoctrineMetadataServiceResourceFactory::class,
-            Model\DoctrineRestServiceModelFactory::class => Model\DoctrineRestServiceModelFactoryFactory::class,
-            Model\DoctrineRestServiceResource::class     => Model\DoctrineRestServiceResourceFactory::class,
-            Model\DoctrineRpcServiceModelFactory::class  => Model\DoctrineRpcServiceModelFactoryFactory::class,
-            Model\DoctrineRpcServiceResource::class      => Model\DoctrineRpcServiceResourceFactory::class,
+            DoctrineAutodiscoveryModel::class      => Model\DoctrineAutodiscoveryModelFactory::class,
+            DoctrineMetadataServiceResource::class => Model\DoctrineMetadataServiceResourceFactory::class,
+            DoctrineRestServiceModelFactory::class => Model\DoctrineRestServiceModelFactoryFactory::class,
+            DoctrineRestServiceResource::class     => Model\DoctrineRestServiceResourceFactory::class,
+            DoctrineRpcServiceModelFactory::class  => Model\DoctrineRpcServiceModelFactoryFactory::class,
+            DoctrineRpcServiceResource::class      => Model\DoctrineRpcServiceResourceFactory::class,
         ],
     ],
     'controllers'                   => [
         // Legacy Zend Framework aliases
         'aliases'   => [
-            DoctrineAutodiscovery::class => Controller\DoctrineAutodiscovery::class,
+            'ZF\Apigility\Doctrine\Admin\Controller\DoctrineAutodiscovery' => 'Laminas\ApiTools\Doctrine\Admin\Controller\DoctrineAutodiscovery',
         ],
         'factories' => [
-            Controller\DoctrineAutodiscovery::class => Controller\DoctrineAutodiscoveryControllerFactory::class,
+            'Laminas\ApiTools\Doctrine\Admin\Controller\DoctrineAutodiscovery' => Controller\DoctrineAutodiscoveryControllerFactory::class,
         ],
     ],
     'api-tools-content-negotiation' => [
         'controllers'            => [
-            Controller\DoctrineAutodiscovery::class   => 'Json',
-            Controller\DoctrineRestService::class     => 'HalJson',
-            Controller\DoctrineRpcService::class      => 'HalJson',
-            Controller\DoctrineMetadataService::class => 'HalJson',
+            'ZF\Apigility\Doctrine\Admin\Controller\DoctrineAutodiscovery'   => 'Json',
+            'ZF\Apigility\Doctrine\Admin\Controller\DoctrineRestService'     => 'HalJson',
+            'ZF\Apigility\Doctrine\Admin\Controller\DoctrineRpcService'      => 'HalJson',
+            'ZF\Apigility\Doctrine\Admin\Controller\DoctrineMetadataService' => 'HalJson',
```

Comment:
These need to be reverted, please. We should only be referring to `Laminas\ApiTools` classes for these sorts of mappings at this point.